pallets/gear-messenger/src/lib.rs

```diff
@@ -591,6 +593,7 @@ pub mod pallet {
         type MailboxSecondKey = MessageId;
         type MailboxedMessage = UserStoredMessage;
         type QueuedDispatch = StoredDispatch;
```

Comment:
It looks like we need to migrate queue, too

Reply:
Migrations contain translation step for message queue (storage `Dispatches`)

Reply:
We are migrating the inner storage of the queue, which is the StoredDispatch if i remember correctly